Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -165,6 +165,92 @@ public void When_an_exception_contains_an_inner_exception_they_should_both_appea
             .And.Contain("InnerExceptionMessage");
     }
 
+    [InlineData(typeof(ulong), "ulong")]
+    [InlineData(typeof(void), "void")]
+    [InlineData(typeof(float?), "float?")]
+    [Theory]
+    public void When_the_object_is_a_primitive_type_it_should_be_formatted_as_language_keyword(Type subject, string expected)
+    {
+        // Act
+        string result = Formatter.ToString(subject);
+
+        // Assert
+        result.Should().Be(expected);
+    }
+
+    [InlineData(typeof(List<FormatterSpecs>), "System.Collections.Generic.List<FluentAssertions.Specs.Formatting.FormatterSpecs>")]
+    [InlineData(typeof(Dictionary<,>), "System.Collections.Generic.Dictionary<TKey, TValue>")]
+    [InlineData(typeof(Dictionary<List<IEnumerable<byte>>, Dictionary<string, Dictionary<int, object>>>), "System.Collections.Generic.Dictionary<System.Collections.Generic.List<System.Collections.Generic.IEnumerable<byte>>, System.Collections.Generic.Dictionary<string, System.Collections.Generic.Dictionary<int, object>>>")]
```

Comment:
Why does this format it like the expected result and not like `System.Collections.Generic.Dictionary<TKey, TValue>` (after https://github.com/AwesomeAssertions/AwesomeAssertions/pull/78/commits/eaae8aec69e379fc7776b1615a1a24b96e117f66)? Am I missing something?